src/main/java/sirius/biz/protocol/DelegateJournalData.java

```diff
@@ -118,6 +122,30 @@ public void setSilent(boolean silent) {
         this.silent = silent;
     }
 
+    /**
+     * Adds an entry to the journal of the given entity.
+     *
+     * @param targetType             the type of the entity under which the journal entity will be written
+     * @param targetId               the id of the entity under which the journal entity will be written
+     * @param contentIdentifierClass the type of the actual entity which was changed
+     * @param contentIdentifierId    the id of the actual entity which was changed
+     * @param changes                the entry to add to the journal, automatically appended by the class's label
+     */
+    public static void createJournalEntry(String targetType,
```

Comment:
use this in  `private void createJournalEntry(Supplier<String> messageSupplier)'  below to removed redundant code?